quipucords/api/source/serializer.py

```diff
@@ -565,11 +567,20 @@ def is_valid_proxy_url_format(proxy_url):
         if not parsed.port or not (MIN_PORT < parsed.port < MAX_PORT):
             raise ValidationError(_(messages.SOURCE_INVALID_PORT_PROXY_URL))
 
+    def _apply_ssl_cert_verify_default(self, validated_data):
+        """Apply default True to ssl_cert_verify if omitted and source is HTTP."""
+        source_type = validated_data.get("source_type")
+        if (
+            source_type in self.HTTP_SOURCE_TYPES
+            and "ssl_cert_verify" not in self.initial_data
+        ):
+            validated_data["ssl_cert_verify"] = True
+
```

Comment:
while not exactly incorrect, this goes against DRF ergonomics.

if for whatever reason I invoke `serializer.validate()`, I should get a dictionary with the validated data.

The way this method is being consumed completely changes that behavior. I'd either have this method being exclusively invoked by `validate` method or just drop the function and add these lines to the validate method

Reply:
I agree that keeping this kind of logic within the validation flow would align better with DRF conventions. I will re-think this.